### PR TITLE
add evm

### PR DESCRIPTION
**Pull request by rlibouba**
Hello, I'm opening this PR to create the conda package of evm (EVidenceModeler) 
Thank you! Have a good day!

### CODE REVIEW

File: recipes/evm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/EVidenceModeler/EVidenceModeler/releases/tag/EVidenceModeler-v{{ version }}.tar.gz"
+  sha256: "{{ sha256 }}"
+
+build:
+  number: 0
+  script: "make"
```
Comment (martin-g) [inline recipes/evm/meta.yaml:15]: I guess you will need to copy some binaries to $PREFIX/bin.
See other recipes' `build.sh` for inspiration!

File: recipes/evm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+requirements:
+  build:
+    - automake
+    - make
+    - gcc
```
Comment (martin-g) [inline recipes/evm/meta.yaml:21]:
```suggestion
    - compiler('c')
```

File: recipes/evm/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    - g++
```
Comment (martin-g) [inline recipes/evm/meta.yaml:22]:
```suggestion
    - compiler('cxx')
```

File: recipes/evm/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    - cmake
+    - perl
+    - python
```
Comment (martin-g) [inline recipes/evm/meta.yaml:25]: perl and python should be moved to the `host` dependencies

**Comment by mencian** [comment]
Hi @rlibouba, a recipe already exists for 'EVidenceModeler' [here](https://github.com/bioconda/bioconda-recipes/tree/master/recipes/evidencemodeler). I s there a reason why another EVM recipe is needed as I believe it would be redundant.

**Comment by rlibouba** [comment]
I hadn't actually seen it. Thank you

**Comment by rlibouba** [comment]
The recipe already exists for EVidenceModeler

**Pull request closed without merging**